bb-aem-integration-tests/src/test/java/com/cognifide/bdd/demo/aem/touchui/Siteadmin62Test.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2016 Cognifide Ltd..
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.cognifide.bdd.demo.aem.touchui;
+
+import static org.junit.Assert.*;
+
+import java.time.LocalDateTime;
+
+import com.cognifide.bdd.demo.GuiceModule;
+import com.cognifide.qa.bb.aem.AemLogin;
+
+import com.cognifide.qa.bb.aem.touch.siteadmin.aem62.SiteadminPage;
+import com.cognifide.qa.bb.aem.touch.siteadmin.common.ActivationStatus;
+import com.cognifide.qa.bb.aem.touch.siteadmin.common.PageModificationInfo;
+import com.cognifide.qa.bb.constants.Timeouts;
+import com.cognifide.qa.bb.junit.Modules;
+import com.cognifide.qa.bb.junit.TestRunner;
+import com.cognifide.qa.bb.provider.selenium.BobcatWait;
+import com.google.inject.Inject;
+import org.hamcrest.CoreMatchers;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+@RunWith(TestRunner.class)
+@Modules(GuiceModule.class)
+public class Siteadmin62Test {
+
+  private static final String CONTEXT_PATH =
+      "/content/geometrixx-outdoors-mobile/en/user/mailbox/sent-items";
+
+  public static final String COMPOSE_MESSAGE_PATH =
+      "/content/geometrixx-outdoors-mobile/en/user/mailbox/compose-message";
+  public static final String DEFAULT_TEMPLATE = "Mobile Product Page";
+
+  @Inject
+  private SiteadminPage siteadminPage;
+
+  @Inject
+  private AemLogin login;
+
+  @Inject
+  private WebDriver driver;
+
+  @Inject
+  private BobcatWait wait;
+
+  private LocalDateTime scheduleTime =
+      LocalDateTime.now().plusMonths(2).plusDays(1).plusMinutes(2).plusMinutes(5);
+
+  @Before
+  public void setUp() {
+    login.authorLogin();
+    siteadminPage.open(CONTEXT_PATH);
+    assertTrue(siteadminPage.isLoaded());
+  }
+
+  @After
+  public void cleanUp() {
+    siteadminPage.open(CONTEXT_PATH);
+    siteadminPage.deleteSubPages();
+    siteadminPage.open(COMPOSE_MESSAGE_PATH);
+    siteadminPage.deleteSubPages();
+  }
+
+  @Test
+  public void shouldOpenSiteadminInDesiredContext() {
+    assertThat(driver.getCurrentUrl(), CoreMatchers.endsWith(CONTEXT_PATH));
+  }
+
+  @Test
+  public void shouldCreateNewPage() {
+    siteadminPage.createNewPage("NewPage", "newpage", DEFAULT_TEMPLATE);
```

Comment:
We should check if page doesn't exist. It happens if page was created and test suddenly stops. The old SiteAdminTest can  be used as example